Documentation/push-templates/android/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Adobe Experience Platform SDK push templates
+description: Learn about the push templates provided and supported by the Adobe Campaign Classic Mobile SDK extension.
+keywords:
+- Adobe Campaign Classic
+- Push
+- Push Template
+- Push Templates
+---
+
+# Push templates setup - Android
+
+This document outlines the steps required to configure your app to use default push templates provided by Adobe.
+
+<InlineAlert variant="info" slots="text"/>
+
+Default push template functionality is available for use with the **Adobe Campaign Classic** extension. <br /><br />This is supported by **Android SDK version 3.1.0+**.
+
+## Setup
+
+### Prerequisite
+
+Add the Campaign Classic extension dependency to your app's gradle file:
+
+```groovy
+dependencies {
+  implementation("com.adobe.marketing.mobile:campaignclassic:3.1.+")
```

Comment:
Do we want to change this to `com.adobe.marketing.mobile:notificationbuilder:3.0.+` instead? Considering this is a stand alone library we should be able to test this without ACC extension